feature - add default texture and copyright in the preference window

btronquo opened the pull request:
Hi, 

- added two optional fields in the preference window of the plugin to set the default Texture location and Copyright name
- updated the version number to **0.41.4**

![image](https://user-images.githubusercontent.com/5455001/139588311-5030840f-43d3-46d0-97c7-481b1ba69b51.png)


**Warning** : if you use the Q menu to export, you'll need to remove the 'export to..' and set again. If not, you'll not able to see the changes

Close #4

tml1024 commented:
Why do you change two `default=''` to `default=""` ? (Honest question, there might indeed be a good reason, maybe that is more "Pythonic", I am far from a Python expert.)

btronquo commented:
That's a good question : In this case I made some tests and replaced the ` '` with `"` maybe because I like more " when we need to put string values in the code than having `'` 
I didn't saw that I had changed that ^^

Anyway, the plugin need a big amount of cleaning work so feel free to replace with default `'` or keep the `'` as you wish :)

tml1024 commented:
I was looking at the "total" end diff of this PR (by appending .diff to the URL). If I look at the individual commits it is clear that it is not intentional. (And adding another commit that just changes the double-quotes to single-quotes because that is how they were initially would be silly.)  I trust this works the way you want and will merge it.

btronquo commented:
Thanks
Can you make a release with patch note or give me the rights to do so
Also, maybe we can create a wiki here since with time the docs will be outdated and hard to maintain

tml1024 commented:
I invited you as a collaborator, is that enough to give you the power you need?

btronquo commented:
Perfect ! Thanks ;)
